src/main/java/net/pms/dlna/RootFolder.java

```diff
@@ -1477,8 +1477,8 @@ public void notify(String filename, String event, FileWatcher.Watch watch, boole
 						RealFile rf = new RealFile(file);
 						rf.setParent(rf);
 						rf.getParent().setDefaultRenderer(RendererConfiguration.getDefaultConf());
-
 						rf.resolveFormat();
+						rf.syncResolve();
```

Comment:
@Nadahar from memory, `syncResolve` is historically only used for certain kinds of media like DVD ISOs. I don't think it is needed, historically. Anyway this is just a bit of a hack to put the file into the database, it's still not the "proper" way of adding a file.